go.mod

```diff
@@ -173,10 +174,10 @@ require (
 	k8s.io/cli-runtime v0.26.0 // indirect
 	k8s.io/client-go v0.26.1 // indirect
 	k8s.io/component-base v0.26.1 // indirect
-	k8s.io/klog/v2 v2.90.1 // indirect
-	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f // indirect
+	k8s.io/klog/v2 v2.100.1 // indirect
+	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9 // indirect
```

Comment:
Hold the version upgrade activity. It will cause the wrong type converting like blow:

```
#18 89.24 /go/pkg/mod/sigs.k8s.io/kustomize/kyaml@v0.13.9/openapi/openapi.go:656:33: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as type *"github.com/google/gnostic-models/openapiv2".Document in argument to swagger.FromGnostic
#18 100.8 # k8s.io/client-go/applyconfigurations/meta/v1
#18 100.8 /go/pkg/mod/k8s.io/client-go@v0.26.1/applyconfigurations/meta/v1/unstructured.go:64:38: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as type *"github.com/google/gnostic-models/openapiv2".Document in argument to proto.NewOpenAPIData
#18 102.4 # k8s.io/kubectl/pkg/util/openapi
#18 102.4 /go/pkg/mod/k8s.io/kubectl@v0.26.0/pkg/util/openapi/openapi.go:52:38: cannot use doc (variable of type *"github.com/google/gnostic/openapiv2".Document) as type *"github.com/google/gnostic-models/openapiv2".Document in argument to proto.NewOpenAPIData
#18 ERROR: executor failed running [/bin/sh -c go mod tidy; CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build 	-ldflags="-w -s -X main.version=$VERSION -X main.gitsha=$GIT_COMMITSHA" 	-a -o meshery-linkerd main.go]: exit code: 1
```